cpp/include/algorithms.h

```diff
@@ -278,3 +278,20 @@ gdf_error gdf_snmg_pagerank (
             const size_t n_gpus, 
             const float damping_factor, 
             const int n_iter);
+/**                                                                             
+ * @Synopsis   Compute the Katz centrality for the nodes of the graph G
+ *                                                                              
+ * @Param[in] *graph                 cuGRAPH graph descriptor with a valid edgeList or adjList
+ *                                                                              
+ * @Param[out] *katz_centrality      If set to a valid column, this is populated by the katz centrality of every vertex in the graph
+ *                                                                              
```

Comment:
Where are the descriptions about the remaining parameters (alpha, max_iter, tol, has_guess, normalized)?

Reply:
Added